plugins/workspace-minimap/src/focus_region.ts

```diff
@@ -190,3 +187,9 @@ export class FocusRegion {
       return this.initialized;
     }
 }
+
+Blockly.Css.register(`
+.focusRegion {
```

Comment:
Looking back at the style guide I think this should also be kebab-case and not camelCase.

Ref: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Style